app-misc/deark/deark-1.6.5.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Utility to extract data from various file formats"
+DESCRIPTION="Deark is a command-line utility that can decode certain types of files, and either:
+
+1. convert them to a more-modern or more-readable format; or
+2. extract embedded files from them"
+HOMEPAGE="https://entropymine.com/deark/
+	https://github.com/jsummers/deark"
+COPYRIGHT="2023 Jason Summers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jsummers/deark/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="2aa29675c0b326c5242da2c5e4b1d7746f2aad7bcc9bfefa73448e580e9fae2c"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	deark$secondaryArchSuffix = $portVersion
+	cmd:deark = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	help2man$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:help2man
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make
+	make man
+}
+
+INSTALL()
+{
+	mkdir -p $prefix/bin
```

Comment:
Are you sure you still need to manually create $prefix/bin (this should be done with line 47)?

Reply:
Yeah, I tried it without and it failed.

```
Collecting files to be packaged ...
install -s deark /packages/deark-1.6.5-1/.self/bin
Warning: [Errno -2147459067] Not a directory: '/packaging/deark/bin'
cleaning 'chroot/boot' folder
keeping chroot folder /boot/home/haikuports/app-misc/deark/work-1.6.5 intact for inspection
Error: Build has failed - stopping.
```